be/src/exec/pipeline/olap_table_sink_operator.h

```diff
@@ -72,8 +79,11 @@ class OlapTableSinkOperatorFactory final : public OperatorFactory {
 
 private:
     std::unique_ptr<starrocks::DataSink> _data_sink;
-    starrocks::stream_load::OlapTableSink* _sink;
+    starrocks::stream_load::OlapTableSink* _sink0;
     FragmentContext* const _fragment_ctx;
+    int32_t _cur_sender_id;
+    size_t _tablet_sink_dop;
+    std::vector<std::unique_ptr<starrocks::stream_load::OlapTableSink>> _sinks;
```

Comment:
multi OlapTableSink use one RuntimeState will have concurrent problem

Reply:
append_error_msg_to_file and append_tablet_commit_infos may have concurrent problem, i update a new pr, it adds lock to avoid this